CMakeLists.txt

```diff
@@ -186,6 +186,11 @@ if (NOT DISABLE_SCRIPTING)
     createExtension(SCRIPTING-EXTENSIONS "SCRIPTING EXTENSIONS" "This enables scripting" "extensions/script" "${TEST_DIR}/script-tests")
 endif()
 
+## SQLite extensions
+option(DISABLE_SQLITE "Disables the scripting extensions." OFF)
```

Comment:
Curious why you chose this to be enabled by default? Because there is no external dependency beyond sqllite?

Reply:
Yeah, basically it has a very portable single-file compile, no deps, etc. I think when users want a small binary they would turn off everything they don't need. Perhaps it would be useful to have a "minimal" flag that covers it all at once.

Reply:
The base is intended to be a minimal build.